diyepw/create_amy_epw_files_for_years_and_wmos.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import os
+from typing import List, Tuple
+from .create_amy_epw_file import create_amy_epw_file
+from ._logging import _logger
+
+def create_amy_epw_files_for_years_and_wmos(
```

Comment:
Based on the function name, I would expect the order of params to be `years` then `wmo_indices`

Reply:
I agree. I swapped them. No other changes are necessary in the project because the calls to this method that exist (in the scripts repo and in tests) use named parameters